schema.sql

```diff
@@ -19,3 +19,14 @@ ALTER TABLE IF EXISTS animals
 
 COMMENT ON TABLE animals
     IS 'A microverse exercise to create an animal table.';
+
+
+-- Column: animals.species
+
+-- ALTER TABLE IF EXISTS animals DROP COLUMN IF EXISTS species;
+
+ALTER TABLE IF EXISTS animals
+    ADD COLUMN species text COLLATE pg_catalog."default";
```

Comment:
- [ ] Kindly note that according to the project requirements, the species column is supposed to be of type string. Kindly change it from text to string.

![image](https://user-images.githubusercontent.com/86318284/189982967-f0a064e5-2f8c-4654-a6a3-a630c29ca097.png)

Reply:
@Nemwel-Boniface Thank you for the review, according to the latest documentation of PostgreSQL the new data types are mentioned here 

https://www.prisma.io/dataguide/postgresql/introduction-to-data-types 

and also mentioned here

https://www.postgresql.org/docs/current/datatype.html. 

As mentioned, the typed string is no longer recognizable in PostgreSQL. As I changed the data type from text to string it indicated that it is not a recognizable type. As you can see in VS Code and also in pgAdmin 4 the option for data type string is not there. 

![options](https://user-images.githubusercontent.com/57537718/190137320-db45a91b-683f-4a72-930f-35bdd2162aaa.png)

![datatype](https://user-images.githubusercontent.com/57537718/190136987-2dbcc932-bfab-4a61-84cf-524fdb39f8ff.png)